tools/packaging/guest-image/build_image.sh

```diff
@@ -37,6 +37,7 @@ build_initrd() {
 	info "initrd os: $os_name"
 	info "initrd os version: $os_version"
 	sudo -E PATH="$PATH" make initrd \
+		VARIANT="${image_initrd_suffix}" \
```

Comment:
Hi @zvonkok !

The other two commits I could understand and they don't need more context. This one I didn't get. Perhaps it will make sense with the following PRs? I'm afraid of getting it merged, devs start using this VARIANT thing in a different context then clashes with your original intention. So maybe either put this in you following PRs or simply stuff this PR with more commits...